web/themes/ventuno/src/scss/components/_homepage.scss

```diff
@@ -47,3 +47,19 @@
   }
 }
 
+// Region Featured  
+.region-featured {
+  background-color: $gray-300;
+  position: relative;
+  overflow: hidden;
+  &:before {
+    content: "";
+    position: absolute;
+    background-color: $gray-100;
+    width: 100%;
+    height: 350%;
+    border-radius: 50%;
+    top: -200%;
+    transform: translate(-50%, 0);
+  }
```

Comment:
This needed a tweak, since this graphical element was overlapping with the info text. I have put a z-index of -1 on this to fix it.